<launch>
  <node pkg="carla_camera_publisher" type="publish_carla_cam.py" name="carla_camera_publisher" output="screen">
    <param name="host" value="localhost"/>
    <param name="port" value="2000"/>
    <param name="image_topic" value="/carla/yolop/image_raw"/>
  </node>
</launch> 